01-js-intro-and-jquery/keypoints/1_11/resources/script/script.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+
+var onPageReady = function () {
+    var onRequestSuccess;
+    var onRequestError;
+    var listLength;
+    var albums;
+    var appendAlbum;
+    var container = $('.container');
+
+    appendAlbum = function (album) {
+        var albumItem = $('<article></article>');
+        var albumName   = $('<h3>Name: ' +album.name+ ' </h3>');
+        var albumType   = $('<h4>Type: '+album.album_type+'</h4>');
+        var albumImage = $('<img src="'+album.images[1].url+'" alt="This image show the photo of album called: '+album.name+'" />');
+
+        // var albumReleaseDate = ""; The guide ask for this attribute but i don't find it.
+        var albumLink = $('<p> <a href=" '+album.external_urls.spotify+'"> Play the music!</a> </p>');
+
+        albumItem.append(albumName,albumType,albumImage,albumLink);
+        container.append(albumItem);
+    }
+
+
+    onRequestSuccess = function (result) {
+
+        listLength = result.albums.items.length;
```

Comment:
I think this variable should be defined inside this function variable instead of outside.